### PR TITLE
refactor: Move to client-side React

### PR DESCRIPTION
**Pull request by Yserz**

### CODE REVIEW

File: .elasticbeanstalk/config.yml [hunk 1]
```diff
@@ -1,10 +1,12 @@
 branch-defaults:
+  react:
```
Comment (Yserz) [inline .elasticbeanstalk/config.yml]: TODO: Removed before merge

File: .travis.yml [hunk 2]
```diff
@@ -37,17 +37,27 @@ after_success:
...
   - |
     if [ -n "$TRAVIS_TAG" ]; then
+      yarn bundle:prod
       if [ "$DISTRIBUTION" == "wire" ]; then
         yarn docker '' production
       else
         yarn docker "$DISTRIBUTION" production
       fi
     fi
+  - |
```
Comment (Yserz) [inline .travis.yml]: TODO: Remove before merge

File: .travis.yml [hunk 3]
```diff
@@ -76,3 +86,19 @@ deploy:
       repo: wireapp/wire-account
       branch: staging
       condition: $DISTRIBUTION = wire
+
+  - provider: elasticbeanstalk
```
Comment (Yserz) [inline .travis.yml]: TODO: Remove before merge

File: tsconfig.json [hunk 4]
```diff
@@ -1,21 +1,24 @@
...
+    "baseUrl": "./src",
     "declaration": false,
     "downlevelIteration": true,
     "forceConsistentCasingInFileNames": true,
     "jsx": "react",
     "lib": ["dom", "es2017"],
     "moduleResolution": "node",
     "noEmitOnError": true,
-    "noImplicitAny": true,
+    "noImplicitAny": false,
```
Comment (ffflorian) [inline tsconfig.json]: Do you need this?

Comment (Yserz) [reply tsconfig.json]: Good one!

**Pull request merged**